src/components/MferHeads.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react'
+import { Row, Card, Nav, Col } from 'react-bootstrap'
+import { createImgObject, fetchMferHead } from '../utils/utils'
+import { useForm } from 'react-hook-form'
+
+const MferHeads = ({ handleSetImage, images }) => {
+	const [mferHeads, setMferHeads] = useState([])
+	const [fetchingHeads, setFetchingHeads] = useState(true)
+	const [mferHeadNumber, setMferHeadNumber] = useState(0)
+	const [mferToFetch, setMferToFetch] = useState(0)
+	const { handleSubmit } = useForm()
+
+	useEffect(() => {
+		const fetchHeads = async () => {
+			const dataURLs = await fetchMferHead()
+			setFetchingHeads(false)
+			setMferHeads(dataURLs)
+		}
+
+		fetchHeads()
+	}, [])
+
+	const makeImgObjectAndHandleSetImage = async dataURL => {
+		const imgObj = await createImgObject(images, dataURL, true)
+		handleSetImage(imgObj)
+	}
+
+	const renderMferHeads = () => {
+		return mferHeads.map((dataURL, idx) => {
+			return (
+				<Col
+					key={`${dataURL}${idx}`}
+					onClick={() => makeImgObjectAndHandleSetImage(dataURL)}
+				>
+					<Card>
+						<Card.Img src={dataURL} />
+						<Card.Body>
+							<Card.Text>This mfer is the is cool as shit</Card.Text>
```

Comment:
does "This mfer is the is cool as shit" need to be hardcoded in? it makes the UI busy, not clear on purpose